web/home/src/components/managePluginCard.tsx

```diff
@@ -0,0 +1,27 @@
+import plusSmall from '../assets/pictos/plusSmall.svg';
+import PlusSmallWhite from '../assets/pictos/PlusSmallWhite.svg';
+import { UIStore } from '../store/UIStore';
+const ManagePluginCard = () => {
+  const plus = UIStore.useState((s) =>
+    s.theme == 'light' ? plusSmall : PlusSmallWhite,
+  );
+  return (
+    <div
+      onClick={() => {
+        window.open('/thyra/plugin-manager');
+      }}
+      className="flex flex-col justify-center items-start p-10 gap-5 w-64 h-72 rounded-2xl"
+    >
+      <button>
+        <img
+          src={plus}
+          alt="plusSmall"
+          className="rounded-full w-16 h-16 mx-auto bg-bgCard hover:bg-hoverbgCard"
+        />
+        <h2 className="label text-font">Manage plugin</h2>
```

Comment:
@SlnPons cc

Reply:
It would make sense to do what you suggest Mario in the next version of the home page when implementing the rebranding (depending on the work evaluation).

Reply:
Yes we will try to implement the internationalisation in the rebranding phase and add more robustness

Reply:
Let's do it together! I'll let you ping me when you want. We start our base for that and when necessary we upgrade for something bigger. We can do better for now with few lines of code. Please, ping me.

Closing here meanwhile! :)